jetty-server/src/test/java/org/eclipse/jetty/server/AsyncCompletionTest.java

```diff
@@ -218,4 +226,44 @@ public void testAsyncCompletion(Handler handler, int status, String message) thr
             }
         }
     }
+
+    private static class AsyncReadyCompleteHandler extends AbstractHandler
+    {
+        static String data = "Now is the time for all good men to come to the aid of the party";
+
+        @Override
+        public void handle(String target, Request baseRequest, HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException
+        {
+            AsyncContext context = request.startAsync();
+            ServletOutputStream out = response.getOutputStream();
+            out.setWriteListener(new WriteListener() {
```

Comment:
Brace at end of line. You are being assimilated.
This whole file needs a formatting pass.